sdks/go/node/containerruntime/docker/runContainer.go

```diff
@@ -190,49 +185,49 @@ func (cr _runContainer) RunContainer(
 		return nil, err
 	}
 
-	containerJSON, err := cr.dockerClient.ContainerInspect(ctx, dockerContainerName)
-	if err != nil {
-		return nil, err
-	}
+	eg, ctx := errgroup.WithContext(ctx)
 
-	if endpointSettings, ok := containerJSON.NetworkSettings.Networks[networkName]; ok && req.Name != nil {
-		err = dns.RegisterName(
-			ctx,
-			*req.Name,
-			endpointSettings.IPAddress,
-		)
+	eg.Go(
```

Comment:
This was unrelated but just cleans up the code to be more readable & less complex.